eastdetector/__init__.py

```diff
@@ -153,39 +166,14 @@ class TextDetector:
     Adapted from: see top
     """
 
-    def __init__(self, checkpoint_path: str):
+    def __init__(self, pb_path: str):
         tf.logging.set_verbosity(tf.logging.ERROR)
 
-        self.__closed = False
-
-        checkpoint_state = tf.train.get_checkpoint_state(checkpoint_path)
-
-        if checkpoint_state is None:
-            raise RuntimeError("Failed to read the tensorflow checkpoint path from: {!r}".format(checkpoint_path))
+        self.pb_path = pb_path
 
-        model_path = os.path.join(checkpoint_path, os.path.basename(checkpoint_state.model_checkpoint_path))  # pylint: disable=no-member
+        self.layer_names = ["feature_fusion/Conv_7/Sigmoid", "feature_fusion/concat_3"]
```

Comment:
Could you please double check that these layer names are always valid? What happens if someone inputs a modified model?